data/human/human missions.txt

```diff
@@ -6893,3 +6893,63 @@ mission "Returning to Paradise"
 			action
 				payment 120000
 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`
+
+mission "Hogshead's Dilemma"
+	invisible
+	source
+		near Sol 1 10
+	to offer
+		random < 5
+	on offer
+		conversation
+			`While passing through the spaceport, you notice a few energetic pilots gathered around a computer terminal, invested in some sort of conversation. One of the pilots speaks louder than the rest, sometimes getting looks from others in the spaceport. "You've fooled me before, and I reckoned you were this time! And here I thought them dang things were old news!" The pilot lets out a hearty chuckle. "Even nowadays them Firkins are still getting names!" After another minute or two, the group walks away from the terminal and out of your sight.`
+			choice
+				`	(Go see what this is about.)`
+				`	(Ignore it.)`
+					decline
+			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads 'Popular Votes For Ship Name Poll - Unexpected Results'. The article is about a small transport company harkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, however it seems a good amount of pilots are eager to join in on the fun.`
```

Comment:
```suggestion
			`	You walk up to the computer terminal, and on it is displayed a news article with a title that reads "Popular Votes For Ship Name Poll - Unexpected Results." The article is about a small transport company harkening back to older times with a special transport voyage centered around a Hogshead space liner. Hogsheads have all but lost the charm they once had, but it seems a good number of pilots are eager to join in on the fun.`
```
Pretty sure this is the way that them quotation marks are used in Yankland.

I believe "number" is preferred instead of "amount" when it comes to countable objects.